### PR TITLE
chore: add background gradient to connectSection social links

### PR DESCRIPTION
**Pull request by haveanicedavid**
## Description

Closes: #regen-network/regen-web#1098

CleanShot 2022-09-08 at 02 54 01@2x

--

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-www/src/components/ConnectSection.tsx [hunk 1]
```diff
@@ -78,17 +78,19 @@ const IconLabel = ({
...
+            background:
+              'linear-gradient(201.8deg, #4FB573 14.67%, #B9E1C7 97.14%);',
             borderRadius: '50%',
             transitionDuration: '200ms',
             transitionProperty: 'color, background-color',
             transitionTimingFunction: 'ease-in-out',
             width: {
               xs: theme.spacing(15),
-              tablet: theme.spacing(30),
+              tablet: 80,
```
Comment (flagrede) [inline web-www/src/components/ConnectSection.tsx:89]: We can't use the theme.spacing util anymore?

File: web-www/src/components/ConnectSection.tsx [hunk 2]
```diff
@@ -78,17 +78,19 @@ const IconLabel = ({
         <Box
           sx={theme => ({
             bgcolor: 'secondary.main',
+            background:
+              'linear-gradient(201.8deg, #4FB573 14.67%, #B9E1C7 97.14%);',
```
Comment (flagrede) [inline web-www/src/components/ConnectSection.tsx:82]: I'm guessing this can't (should not ?) be added to the theme.

**Review by flagrede — APPROVED** [review]
Looks good, just a few questions.

**Review by victordelval — APPROVED** [review]

File: web-www/src/components/ConnectSection.tsx (hunk 1 above)
Comment (haveanicedavid) [reply web-www/src/components/ConnectSection.tsx:89]: actually an oversight - good catch

File: web-www/src/components/ConnectSection.tsx (hunk 2 above)
Comment (haveanicedavid) [reply web-www/src/components/ConnectSection.tsx:73]: it’s a gradient specific to round buttons like this, so probably not worth having in the theme. 

One pattern I like to do in other apps is something like a `styles.constants.ts` file, and could have something like 
```ts
GRADIENTS = {
  round: ''linear-gradient(201.8deg, #4FB573 14.67%, #B9E1C7 97.14%);’'
  button: ..etc
  }
```

but I didn’t want to make that sort of change to the style of the codebase without talking. Doing it here seems more in line with current patterns (though I could extract it to a shared compoonent, but figured we would do that if we reuse anywhere)

**Comment by haveanicedavid** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**